### DISCUSSION TITLE
mod_dav_svn on MacOS for apache and subversion intergration

### DISCUSSION THREAD

**Discussion by youngho1203** [Homebrew/discussions#2840] [Everyday usage]
Hello,
I would like to use subversion with apache on Mac ( Intel mac mini ).
previously I used mod_dav_svn.so and mod_authz_svn.so. 
but when I use brew installed version apache and subversion, I can not find how to get those two binary file.
Any help ?

**Comment by SMillerDev** [comment]
You'll need to compile your own version of apache since brew does not include these.

**Comment by youngho1203** [comment]
I am tring to install
brew install -s httpd with ./configure --enable-dav --enable-so option.
how can I configure with --enable-dav --enable-so option ?

**Reply by youngho1203** [reply]
Can I use httpd.rb ( https://github.com/Homebrew/homebrew-core/blob/HEAD/Formula/httpd.rb )
just add  '--enable-dav --enable-so' at the ./configure part and build interactive and from source
How can I do that ?
